swagger_to_sdk_config.json

```diff
@@ -4,10 +4,9 @@
       "autorest_options": {
         "az": "",
         "use": "@autorest/az@latest",
-        "sdkrel:az-src-folder": "./src/.",
-        "version": "3.0.6198",
-        "sdkrel:output-folder":".",
-        "clear-output-folder":"false"
+        "version": "3.0.6271",
```

Comment:
is it on purpose to remove `output-folder`?

Reply:
yes. the latest @autorest/az package 1.3.2 use --azure-cli-extension-folder instead of --output-folder to specify the output location. @autorest/az will add the src/extension-name.
it also allows the --clear-output-folder except manual folder.
Futher info can found here.
https://github.com/Azure/autorest.az/blob/master/doc/faq.md#what-would-the-clear-output-folder-clear
But please don't merge it now. @tiramisuq still needs some change in the swagger pipeline to make the automation work.

Reply:
sure. Ping me when this PR is ready to merge.